src/Features/Lsif/Generator/Writing/ILsifJsonWriter.cs

```diff
@@ -2,12 +2,14 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System.Collections.Generic;
 using Microsoft.CodeAnalysis.LanguageServerIndexFormat.Generator.Graph;
 
 namespace Microsoft.CodeAnalysis.LanguageServerIndexFormat.Generator.Writing
 {
     internal interface ILsifJsonWriter
     {
         void Write(Element element);
+        void Write(List<Element> elements);
```

Comment:
My only apprehension is it seems like one of these methods will write this:

```
{ object }
```

While the other will write this:

```
[ { object }, { object }, ... ]
```

Consider renaming the second one to `WriteAll` to avoid the impression that it writes a single array element and not a set of object elements.

Reply:
gotcha.  will rename.